wp-content/mu-plugins/pantheon/pantheon-try-gutenberg-mods.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+  * Should we proceed with adding the
+  * return to Pantheon button?
+  *
+  * Only if:
+  * We are on a Pantheon subdomain and
+  * "RETURN_TO_PANTHEON_BUTTON" is not false
+  */
+$mod_try_gutenberg = apply_filters( 'show_pantheon_try_gutenberg_mods', true);
+
+function _pantheon_try_gutenberg_host_link(){
+?>
+<style type="text/css">
+    .try-gutenberg-panel .try-gutenberg-panel-column:not(.try-gutenberg-panel-image-column) {
+        grid-template-rows: auto;
+    }
+    .try-gutenberg-panel .host-link {
+        font-weight: bold;
+    }
+</style>
+<p>
+<?php
+printf(
+ 	/* translators: Link to https://pantheon.io/gutenberg */
+ 	__( '<a href="%s" class="host-link">Learn about Gutenberg on Pantheon</a>' ),
+ 	__( 'https://pantheon.io/gutenberg' )
+);
+?>
+</p>
+<?php    
+}
+
+if( true === $mod_try_gutenberg ){
+    add_action( 'try_gutenberg_after_install_button', '_pantheon_try_gutenberg_host_link');
+}
+
+function _pantheon_try_gutenberg_append_warning(){
+?>
+<style type="text/css">
+    .try-gutenberg-panel .pantheon-notice {
+        color: #2C3539;
+        border-left: solid 5px #2C3539;
+        padding: 0 1em;
+        margin: 0.5em 0;
+    }
+</style>
+<div class="try-gutenberg-panel-content"><div class="pantheon-notice">
```

Comment:
<div class="pantheon-notice">
<h3>Using Gutenberg with Pantheon</h3><p>Make sure Gutenberg is installed in the Dev environment. Use <a href="" target="_blank">the Pantheon workflow</a> to deploy it to your Test and Live environments.</p>
<p>Pantheon offers resources and webinars on getting the most out of Gutenberg. <a href="" target="_blank">Learn about Gutenberg on Pantheon</a>.
</p></div>

Feel free to adjust the copy more, just trying to consolidate Pantheon actions into this one panel. Trying to convey:
1. here's the thing you need to keep in mind for using Gutenberg on Pantheon
2. here's awesome resources that will help you with Gutenberg on Pantheon